layers/factory.go

```diff
@@ -5,10 +5,22 @@ import (
 	"path/filepath"
 	"strings"
 
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+
 	"github.com/buildpacks/lifecycle/archive"
 	"github.com/buildpacks/lifecycle/log"
 )
 
+const (
+	BuildpackLayerName      = "Layer: '%s', Created by buildpack: %s"
+	ExtensionLayerName      = "Layer: '%s', Created by extension: %s"
+	LauncherConfigLayerName = "Buildpacks Launcher Config"
+	LauncherLayerName       = "Buildpacks Application Launcher"
+	ProcessTypesLayerName   = "Buildpacks Process Types"
+	SBOMLayerName           = "Software Bill-of-Materials"
+	SliceLayerName          = "Application Slice: %d"
```

Comment:
Probably not worth it - but I would support having `Application Layer` if you only have a single slice.